presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -324,7 +324,7 @@ default void finishDelete(ConnectorSession session, ConnectorTableHandle tableHa
     /**
      * Create the specified view. The data for the view is opaque to the connector.
      */
-    default void createView(ConnectorSession session, SchemaTableName viewName, String viewData, boolean replace)
+    default void createView(ConnectorSession session, SchemaTableName viewName, String viewData, boolean replace, Optional<String> comment)
```

Comment:
Keep the replace flag at the end. I'm also thinking we should make this take ConnectorViewDefinition instead.

Reply:
Makes sense. In that way, the parameters we pass in will be more structured, and the pattern will be more like ConnectorTableMetadata and consistent.